public/admin/config.yml

```diff
@@ -93,23 +93,6 @@ collections:
             required: false
             hint: Heading banner on top of Home page
             tagname: ""
-      - name: officially_supported_apps
-        label: Officially Supported Apps by Kemenkes
-        file: _content/officially_supported_apps.json
-        fields:
-          - label: Application
-            name: application
-            widget: list
-            fields:
-              - label: Application Name
-                name: name
-                widget: string
-              - label: Download Link
-                name: link
-                widget: string
-              - label: Application Logo
-                name: logo
-                widget: image
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!